app/controllers/hub/sites_controller.rb

```diff
@@ -0,0 +1,47 @@
+module Hub
+  class SitesController < ApplicationController
+    include AccessControllable
+    before_action :require_sign_in
+    before_action :load_organizations, only: [:new, :edit]
+
+    load_and_authorize_resource :vita_partner, parent: false
+
+    layout "admin"
+
+    def new
+      @site = VitaPartner.new(parent_organization_id: params[:parent_organization_id])
```

Comment:
I think that CanCanCan might be initializing a new object called @vita_partner secretly... might make more sense to just authorize_resource vita_partner for new and create since we don't need the objects they instantiate on load!

Reply:
I couldn't get this proposed change to work, and I am going to punt on it.

I do have a remaining question, though: `vita_partner_params` does a `require(:vita_partner)` - so... how can it possibly apply to the `new` method?